Signal/Images.xcassets/group-avatar.imageset/Contents.json

```diff
@@ -2,15 +2,17 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "empty-group-avatar.png",
+      "filename" : "group-28-white@1x.png",
       "scale" : "1x"
     },
     {
       "idiom" : "universal",
+      "filename" : "group-28-white@2x.png",
       "scale" : "2x"
     },
     {
       "idiom" : "universal",
+      "filename" : "group-28-white@3x.png",
```

Comment:
Group avatars - including the "default" group avatar - are now rendered dynamically using this asset.